report.go

```diff
@@ -0,0 +1,21 @@
+package xendit
+
+import "time"
+
+type Report struct {
+	ID         string     `json:"id"`
+	Type       string     `json:"type"`
+	Filter     Filter     `json:"filter"`
+	Format     string     `json:"format"`
+	Status     string     `json:"status"`
+	Url        string     `json:"url"`
+	Currency   string     `json:"currency"`
+	BusinessID string     `json:"business_id"`
+	Created    *time.Time `json:"created"`
```

Comment:
I believe these (`Created` & `Updated`) are both non-nullable required parameters. Could we change the type from `*time.Time` -> `time.Time`

Reply:
Sure, didn't realize it was non-nullable. Will update it soon.